js/libs/form.js

```diff
@@ -211,42 +211,15 @@ Fliplet.Widget.instance('form-builder', function(data) {
               }
               break;
 
-            case 'flCheckbox':
-              if (fieldData.length > 0) {
-                var inOptions = [];
-
-                fieldData.forEach(function (element) {
-                  var match = _.find(field.options, function (option) {
-                    return option.label === element || option.id === element;
-                  });
-
-                  if (match) {
-                    inOptions.push(match);
-                  }
-                });
-
-                field.value = inOptions.length ? _.uniqWith(fieldData, _.isEqual) : [];
-              } else {
-                field.value = [];
-              }
-              break;
-
             case 'flStarRating':
               field.options = _.times(5, function(i) {
                 return {
                   id: '' + (i+1)
                 };
               });
 
-            case 'flRadio':
-              // Work only if passed value is a string
-              // Altered check to support old version of the form builder and if a user provides data as ID not a Label it will work correctly
-              var match = _.find(field.options, function (option) {
-                return option.label === fieldData || option.id === fieldData;
-              });
-
-              field.value = match ? fieldData : '';
-              break;
+            case 'flCheckbox': // There is no validation and value assignment as there is no access to checkbox options. This is implemented in the checkbox component.
+            case 'flRadio': // There is no validation and value assignment as there is no access to radio options. This is implemented in the radio component.
```

Comment:
@OleksandrLavrynenko Currently this says checkbox and radio will also execute this line: `field.value = fieldData;` Is that intentional or do we want to stop the code execution?

```suggestion
            case 'flRadio': // There is no validation and value assignment as there is no access to radio options. This is implemented in the radio component.
              break;
```

Reply:
> Is that intentional

**@tonytlwu**  This is intentional because we need to get the values